hudi-common/src/main/java/org/apache/hudi/common/table/timeline/TimelineMetadataUtils.java

```diff
@@ -70,18 +70,20 @@ public static HoodieRollbackMetadata convertRollbackMetadata(String startRollbac
     Map<String, HoodieRollbackPartitionMetadata> partitionMetadataBuilder = new HashMap<>();
     int totalDeleted = 0;
     for (HoodieRollbackStat stat : rollbackStats) {
-      Map<String, Long> appendFiles = stat.getCommandBlocksCount().keySet().stream()
+      Map<String, Long> rollbackLogFiles = stat.getCommandBlocksCount().keySet().stream()
+          .collect(Collectors.toMap(f -> f.getPath().toString(), FileStatus::getLen));
+      Map<String, Long> probableLogFiles = stat.getProbableLogFileToSizeMap().keySet().stream()
```

Comment:
lets stick to one consistent term please

Reply:
I think you want to capture the fact that this is all the log files. lets just call it `allLogFiles` then